shop/src/Assets/Icons/MastercardIcon.jsx

```diff
@@ -0,0 +1,27 @@
+export default function LoginIcon() {
```

Comment:
MasterCardIcon?

Reply:
It is my fault, at those time, I copied. I forgot to rename it.